docs/concepts/project/dataset.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Datasets"
+id: dataset
+description: Schema, meet Data.
+sidebar_position: 3
+tags:
+  - concepts
+  - datasets
+  - source
+  - target
+---
+
+In Prophecy, Datasets are grouped by [Projects](docs/concepts/project/project.md) and rely on the following:
+
+- **Schema**: The structure or shape of the data, including column names, data types, and the method for reading and writing the data in this format.
+- **[Fabric](docs/concepts/fabrics/fabrics.md)**: The execution environment in which the data resides.
+
+## Create Datasets
+
+Datasets are created where they are first used in a [Source or Target Gems](docs/Spark/gems/source-target/source-target.md). A Dataset definition includes its:
+
+- **Type**: The type of data you are reading/writing like CSV, Parquet files or catalog tables.
+- **Location**: The location of your data. It could be a file path for CSV or a table name.
+- **Properties**: Properties consists of Schema and some other attributes specific to the file format. For example, in case of CSV, you can give Column delimiter in additional attributes. You can also define Metadata for each column here like description, tags, and mappings.
+
+Datasets can be used by any Pipeline within the same Project, and in some cases by other Projects within the same team.
+
```

Comment:
:::info
For regular usage, we suggest having only one instance of a particular dataset within a pipeline, as the dataset's properties and underlying data can change each time the dataset is read / written.
:::